skflow/estimators/base.py

```diff
@@ -240,12 +241,13 @@ def _predict(self, X):
             raise NotFittedError()
         predict_data_feeder = setup_predict_data_feeder(X)
         preds = []
+        dropouts = tf.get_collection(DROPOUTS)
+        feed_dict = {prob: 0.0 for prob in dropouts}
```

Comment:
I think this is not correct for two reasons:
- ``prob`` should be 1.0, not 0.0 at test time (this is ``keep_prob``)
-  right now ``dropouts`` is always empty, because ``self._graph`` is not a default graph

So one way to fix these two lines is to write
```
dropouts = self._graph.get_collection(DROPOUTS) 
feed_dict = {prob: 1.0 for prob in dropouts}    
```